### PR TITLE
Add new metadata properties.

### PR DESCRIPTION
**Pull request by jtherrmann**
Add `version`, `weather_model`, `temporal_baseline_days`, `frame_number`.

TODO:

- [x] Should `version` include any validation to ensure it's a valid version number?
- [x] Make `version` required
- [x] Should `temporal_baseline_days` have a minimum value of `6`? (see https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/pull/102/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR28)
- [x] Should `frame_number` allow a value of `-1`? (see https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/pull/102)
- [x] Confirm what metadata without `weather_model` looks like

### CODE REVIEW

File: verify/src/metadata_schema.json [hunk 1]
```diff
@@ -123,6 +123,28 @@
         },
         "perpendicular_baseline": {
           "type": "number"
+        },
+        "version": {
```
Comment (asjohnston-asf) [inline verify/src/metadata_schema.json]: reviewing past .json files in our archive, they appear to all have `version`. We can add `version` to `required`, above.

File: verify/src/metadata_schema.json [hunk 2]
```diff
@@ -123,6 +123,28 @@
...
+          "type": "string"
+        },
+        "weather_model": {
+          "type": "string",
+          "enum": [
+            "None",
```
Comment (asjohnston-asf) [inline verify/src/metadata_schema.json]: It's unclear to me what the JPL metadata file will look like for v3 files that did not use a weather model at all, this assumes `weather_model: ["None"]`, but that's worth confirming.

File: verify/src/metadata_schema.json [hunk 3]
```diff
@@ -123,6 +124,28 @@
...
+            "HRRR"
+          ]
+        },
+        "temporal_baseline_days": {
+          "type": "integer",
+          "minimum": 1
+        },
+        "frame_number": {
+          "type": "integer",
+          "minimum": 1
```
Comment (asjohnston-asf) [inline verify/src/metadata_schema.json]: I'm not sure if `frame_number` will be zero indexed. I expect there's a valid `maximum` as well but I'm fine moving ahead without that right now.

File: verify/src/metadata_schema.json [hunk 4]
```diff
@@ -123,6 +124,30 @@
         },
         "perpendicular_baseline": {
           "type": "number"
+        },
+        "version": {
+          "type": "string"
+        },
+        "weather_model": {
+          "type": "array",
```
Comment (asjohnston-asf) [inline verify/src/metadata_schema.json:132]: Discussion in slack settled on not including this attribute at all when no weather model was used, so we can enforce that the list is non-empty when a weather model *is* used.
```suggestion
          "type": "array",
          "minItems": 1,
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0]
+### Added
+- One required metadata property: `version`
```
Comment (jtherrmann) [inline CHANGELOG.md]:
```suggestion
- One required top-level property: `version`
```

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**